### DISCUSSION TITLE
Is Cloudesire still working?

### DISCUSSION THREAD

**Discussion by kaiohenricunha** [upptime/upptime#975] [Q&A] [answered]
I was interested on this Helm chart commercialization feature, but couldn’t not figure from the doc alone how to actually use it:

https://docs.cloudesire.com/docs/modules-helm.html

**Comment by AnandChowdhary** [comment, marked as answer]
I think this is the wrong place to ask this question, but please let me know if I'm mistaken. Upptime is an open source status page and uptime monitor, unrelated to Cloudesire.
